ietf/templates/doc/mail/ballot_writeup.txt

```diff
@@ -1,11 +1,12 @@
-{% autoescape off %}
-Technical Summary
-
+{% load ietf_filters %}{% autoescape off %}Technical Summary
+{% if doc.abstract %}
+{{ abstract }}
```

Comment:
`{{ doc.abstract.rstrip }}` should work here, saving the need for the manually split-out argument to the template.

Reply:
Will do, thanks.